4.3: cleanup install-vm.adoc re-using phrases from the new section on proxy vm (#3097)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/install-vm.adoc

[[install-vm]]
= Install {productname} in a Virtual Machine Environment Using {productname} VM Image

[[quickstart.sect.kvm.settings]]
== Virtual Machine Manager Settings

This chapter provides the required Kernel Virtual Machine (KVM) settings for {productname}.
KVM combined with Virtual Machine Manager (_virt-manager_) will be used as a sandbox for this installation.

You will find the VM images for {productname} {productnumber} in various formats.

> It includes {sles} {sles-version} {sp-base} as the underlying operating system and the {productname} software current at the time of build.

Reviewer: ```suggestion
They include {sles} {sles-version} {sp-base} as the underlying operating system and the {productname} software current at the time of build.
```

> === Set root password using [systemitem]``Ignition``

Reviewer: ```suggestion
=== Set Root Password Using [systemitem]``Ignition``
```

[systemitem]``Cloud Init`` uses few configuration files in the YAML format. Used files are named [path]``meta-data``, [path]``network-config`` and [path]``user-data``.

> [systemitem]``Cloud Init`` allows numerous sources where to store configuration data. In this guide we use local iso image with volume id [literal]``cidata`` as a source. The directory structure must look as follows:

Reviewer: ```suggestion
[systemitem]``Cloud Init`` allows numerous sources where to store configuration data. 
In this guide we use local iso image with volume id [literal]``cidata`` as a source. 
The directory structure must look as follows:
```

> === Set up root password using Cloud Init

Reviewer: ```suggestion
=== Set Up Root Password Using Cloud Init
```

You need to hash the root password, for example, by using the [command]``openssl`` command:

----
openssl passwd -6
----

> The command creates a hash of the password you chose. Use this hash as the value of the [literal]``password`` attribute.

Reviewer: ```suggestion
The command creates a hash of the password you chose.
Use this hash as the value of the [literal]``password`` attribute.
```

. Power on the VM, and log in using credentials set by configuration disk.

[[minimal.susemgr.prep]]

> == Preparing virtual machine for {productname}

Reviewer: ```suggestion
== Preparing Virtual Machine for {productname}
```
